lib/shopify_app/session/session_repository.rb

```diff
@@ -2,21 +2,23 @@ module ShopifyApp
   class SessionRepository
     class ConfigurationError < StandardError; end
 
+    EXPECTED_METHODS = [:store, :retrieve].freeze
+    EXPECTED_SHOP_STORAGE_METHODS = (EXPECTED_METHODS + [:retrieve_by_shopify_domain]).freeze
+    EXPECTED_USER_STORAGE_METHODS = (EXPECTED_METHODS + [:retrieve_by_shopify_user_id]).freeze
```

Comment:
Yeah sorry as I just mentioned, I guess I blanked on this and forgot my other work but this will all have to be removed. I am removing it in https://github.com/Shopify/shopify_app/pull/925 as it conflicts with reloading

Reply:
@tanema just updated to account for your changes. Could you give it a quick look? I tophatted the changes.